bitsail-connectors/connector-elasticsearch/src/main/java/com/bytedance/bitsail/connector/elasticsearch/base/NetUtil.java

```diff
@@ -24,7 +24,7 @@
 @Slf4j
 public class NetUtil {
 
-  private static final String IPV4_HTTP_PORT_FORMAT = "([.0-9]*):(\\d+)";
```

Comment:
why we change here?

Reply:
When address uses alias instead of IP (x.x.x.x), _e.g._,  "localhost:1234", this regexp passes "whether match" check, but cannot return right host (it returns null).
```java
String hostPort = "localhost:1234";
boolean matched = NetUtil.isIpv4Address(hostPort);   // return true
String host = NetUtil.getIpv4Ip(hostPort);    // return null
```

Reply:
will the new one match ipv6 as well?

Reply:
It's a problem. 
I use another way to recognize ipv4/ipv6 address, you can see the updated codes.

Reply:
https://github.com/bytedance/bitsail/pull/338/commits